### PR TITLE
#10861 Multi Card Workflow Component and String Structure

### PR DESCRIPTION
**Pull request by csmith-he**
### Types of changes

-   [X] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

In multi card step, the re-population now handles non-observables (as produced with regard to the internationalized structure of the string node)

### Issues Solved
#10861 

### Checklist

-   I targeted one of these branches:
    - [ ] dev/7.6.x (under development): features, bugfixes not covered below
    - [X] dev/7.5.x (main support): regressions, crashing bugs, security issues, major bugs in new features
    - [ ] dev/6.2.x (extended support): major security issues, data loss issues
-   [ ] I added a changelog in arches/releases
-   [ ] I submitted a PR to arches-docs (if appropriate)
-   [ ] Unit tests pass locally with my changes
-   [ ] I added tests that prove my fix is effective or that my feature works
-   [ ] My test fails on the target branch

#### Ticket Background
*   Sponsored by: Historic England
*   Found by: @csmith-he
*   Tested by: @csmith-he

### Further comments

This was going to be submitted against 7.6.x but, as bugs are being submitted against 7.5.x in preparation for Arches for HERs, I am now submitting for 7.5.3

### CODE REVIEW

File: arches/app/media/js/views/components/workflows/workflow-component-abstract.js [hunk 1]
```diff
@@ -533,7 +533,16 @@ define([
...
             Object.keys(tile.data).forEach(function(key) {
                 if (ko.isObservable(tile.data[key])) {
-                    tile.data[key](data.data[key]());
+                    if (ko.isObservable(data.data[key])) {
+                        tile.data[key](data.data[key]());
+                    }
+                    else{
+                        var dataNotObservable = ko.observable(ko.mapping.toJS(data.data[key]));
+                        tile.data[key](dataNotObservable());
+                    }
```
Comment (chiatt) [inline arches/app/media/js/views/components/workflows/workflow-component-abstract.js:542]: `dataNotObservable` gets immediately evaluated on line 541, so it doesn't actually need to be cast to an observable. This block could be simplified a bit like this:
```
if (ko.isObservable(data.data[key])) {
    tile.data[key](data.data[key]());
}
else{
    tile.data[key](ko.mapping.toJS(data.data[key]));
}
```

File: arches/app/media/js/views/components/workflows/workflow-component-abstract.js [hunk 2]
```diff
@@ -533,7 +533,16 @@ define([
...
+
+
+
```
Comment (chiatt) [inline arches/app/media/js/views/components/workflows/workflow-component-abstract.js:545]: It would be good to remove these extra lines as well.

**Review by chiatt — CHANGES_REQUESTED** [review]
This resolves #10861. Just a couple of really minor changes would tighten this up a bit.

File: arches/app/media/js/views/components/workflows/workflow-component-abstract.js (hunk 2 above)
Comment (csmith-he) [reply arches/app/media/js/views/components/workflows/workflow-component-abstract.js]: @chiatt These should now be removed

File: arches/app/media/js/views/components/workflows/workflow-component-abstract.js (hunk 1 above)
Comment (csmith-he) [reply arches/app/media/js/views/components/workflows/workflow-component-abstract.js]: @chiatt Changes made as requested

**Review by chiatt — APPROVED** [review]

**Pull request merged**